newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -1,92 +1,331 @@
 // @flow
-import { Trans } from '@lingui/macro';
+import { t, Trans } from '@lingui/macro';
 import { I18n } from '@lingui/react';
-import { Card, CardActions, CardHeader, Chip } from '@material-ui/core';
 import * as React from 'react';
+import { type I18n as I18nType } from '@lingui/core';
+
+import { Card, Chip } from '@material-ui/core';
+import ShareIcon from '@material-ui/icons/Share';
+import MoreVert from '@material-ui/icons/MoreVert';
+
 import { Column, Line, Spacer } from '../UI/Grid';
 import RaisedButton from '../UI/RaisedButton';
-import { getGameUrl, type Game } from '../Utils/GDevelopServices/Game';
 import { ResponsiveLineStackLayout } from '../UI/Layout';
-import Window from '../Utils/Window';
+import FlatButton from '../UI/FlatButton';
+import IconButton from '../UI/IconButton';
+import Text from '../UI/Text';
+import Toggle from '../UI/Toggle';
+import CardContent from '../UI/Card/CardContent';
+import CardHeader from '../UI/Card/CardHeader';
+import ElementWithMenu from '../UI/Menu/ElementWithMenu';
+
 import { GameThumbnail } from './GameThumbnail';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import ShareDialog from './ShareDialog';
+
+import {
+  getGameUrl,
+  updateGame,
+  type Game,
+} from '../Utils/GDevelopServices/Game';
+import Window from '../Utils/Window';
+import { type GamesDetailsTab } from './GameDetailsDialog';
+import Dialog from '../UI/Dialog';
+import PlaceholderLoader from '../UI/PlaceholderLoader';
+import { showErrorBox } from '../UI/Messages/MessageBox';
 
 type Props = {|
   game: Game,
   isCurrentGame: boolean,
-  onOpenGameManager: () => void,
+  onOpenGameManager: (tab: GamesDetailsTab) => void,
+  onUpdateGame: () => Promise<void>,
 |};
 
-export const GameCard = ({ game, isCurrentGame, onOpenGameManager }: Props) => {
+type TogglableProperties =
+  | 'discoverable'
+  | 'acceptsBuildComments'
+  | 'acceptsGameComments';
+
+const confirmationMessage = {
+  discoverable: {
+    true: <Trans>Are you sure you want to make this game discoverable?</Trans>,
```

Comment:
let's take the opportunity to add a bit more info to these messages, to educate users how this work.
This could become:
`You're about to make this game discoverable on categories pages on Liluo.io. Continue?`